src/main/java/ch/njol/skript/expressions/ExprMaxFreezeTicks.java

```diff
@@ -0,0 +1,63 @@
+
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.util.Timespan;
+import org.bukkit.entity.Entity;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Maximum Freeze Time")
+@Description("The maximum amount of time an entity can spend in powdered snow before taking damage.")
+@Examples({
+	"difference between player's freeze time and player's max freeze time is less than 1 second:",
+	"\tsend \"you're about to freeze!\" to the player"
+})
+@Since("INSERT VERSION")
+public class ExprMaxFreezeTicks extends SimplePropertyExpression<Entity, Timespan> {
+
+	static {
+		if (Skript.methodExists(Entity.class, "getMaxFreezeTicks"))
+			register(ExprMaxFreezeTicks.class, Timespan.class, "max[imum] freeze time", "entities");
+	}
+
+	@Override
+	protected String getPropertyName() {
+		return "maximum freeze time";
+	}
+
+	@Override
+	public @Nullable Timespan convert(Entity entity) {
```

Comment:
Following code conventions
https://github.com/SkriptLang/Skript/blob/2c0b88b13eacc2a3bb50e086664a2877e79d6862/code-conventions.md?plain=1#L61

```suggestion
	@Nullable
	public Timespan convert(Entity entity) {
```

Same goes for the others